src/main/java/io/eroshenkoam/xcresults/export/Allure2ExportFormatter.java

```diff
@@ -80,7 +80,8 @@ public TestResult format(final ExportMeta meta, final JsonNode node) {
         if (Objects.nonNull(result.getStart())) {
             if (node.has(DURATION)) {
                 final Double durationText = node.get(DURATION).get(VALUE).asDouble();
-                result.setStop(result.getStart() + TimeUnit.SECONDS.toMillis(durationText.longValue()));
```

Comment:
Fix test run that take lesser than 1 second will show as 0 second in allure report.